04をやりました

kanakikuchi1110 opened the pull request:
間違えてpushしてしまいました・・・

Commit:
04をやりました

Commit:
クラス命名を修正しました

kanakikuchi1110 commented:
またもや業務中にすみません🙏
04もやりました

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -0,0 +1,67 @@
+  </header>
+  <main>
+    <article>
+      <div class="postHeader">
```
divではなく `header` で良いのでは？

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -0,0 +1,67 @@
+        <img class="postImgFull" src="img/italy.png" alt="italy">
+        <p>本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト本文テキスト</p>
+
+        <ul class="list_number">
```
数字付きのリストは `ul` ではなく `ol` を使いましょう！

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -0,0 +1,67 @@
+          <li>・リスト３</li>
+        </ul>
+
+        <p class="quote">ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。</p>
```
引用の際は `p` タグではなく別のタグを使用します！調べてみましょう 🤔

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -0,0 +1,67 @@
+    </article>
+    </main>
+    <aside class="author">
+      <img src="author.png" alt="author">
```
`alt` タグは具体的なアイテム名を記したほうが良いです。

このままだと、例えば目の不自由な方などが音声ブラウザで聞いたときに
「ここには `author` という画像が入っているのか」
と認識することになります。検索エンジンもそのように解釈します。

となるとどのようなaltタグが適切でしょうか？

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -0,0 +1,67 @@
+    <aside class="author">
+      <img src="author.png" alt="author">
+      <h3>この記事を書いた人</h3>
+      <p class="author_name">山田太郎</p>
```
ここまで命名規則がキャメルケース（ `headerLogo` ）だったのに突然スネークケース（ `header_logo` ）になっています。
サイト内では統一したほうが良いでしょう :)

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -0,0 +1,67 @@
+      <p class="author_caption">フリーライター。いろんなことを書いています。いろんなことを書いています。</p>
+    </aside>
+    <nav class="linear_navigation">
+      <span class="btnPrevious">前の記事</span>
```
このボタンをクリックして前の記事・次の記事に移動するならspanタグで良いでしょうか…？

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -0,0 +1,67 @@
+      <p class="author_name">山田太郎</p>
+      <p class="author_caption">フリーライター。いろんなことを書いています。いろんなことを書いています。</p>
+    </aside>
+    <nav class="linear_navigation">
```
`linear_navigation` という命名に違和感を感じますね… 🤔

OZPA reviewed (commented):
基本的なhtmlの構造や、HTML5タグの扱いが非常に良い感じです！

チェックをいれましたので修正お願いします〜 :)

Commit:
Update index.html

postHeaderのdivをheaderに修正しました。
olに修正してolとulの不要なクラス取りました。
全ての画像のaltを日本語にして、内容も分かりやすくしました。
qタグに修正しました。
リニアナビゲーションのクラス名も変更しました（これでいいのでしょうか？）
spanからbuttonに変更しました。
コピーライトのところのpタグが抜けていたのでいれました。

Commit:
Update index.html

olとulにクラスを付けました。
あとナビゲーションは記事のナビゲーションなのでpostを付けました。

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -15,31 +15,31 @@ <h1 class="headerTitle">サイトタイトル</h1>
         </ul>
 
-        <p class="postQuote">ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。</p>
+        <q>ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。ここは他サイトからの引用テキストです。</q>
```
classは残しておいたほうが良いですね！

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -48,20 +48,22 @@ <h2 class="postTitle">Article Title</h2>
+  </main>
+
+  <aside class="author">
+    <img src="author.png" alt="筆者の顔">
```
惜しい！
altの正解は `山田太郎` が良いですね。

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -48,20 +48,22 @@ <h2 class="postTitle">Article Title</h2>
+
   <footer>
-    Copyright ©️ OZPA All rights reserved.
+    <p>Copyright ©️ OZPA All rights reserved.</p>
```
一応コピーライトもｚclass振っておきましょうか

OZPA commented on Desktop/myworks/temp/html-css-study/04/index.html:
```diff
@@ -48,20 +48,22 @@ <h2 class="postTitle">Article Title</h2>
+  </aside>
+
+  <nav class="linearNav">
+    <a href="previouspage.html"><button type="button" class="btnPrevious">前の記事</button></a>
```
aタグ内にbuttonタグを内包することはできません！
前回はspanタグのみだったので、それがNGという意味でした。

正しくは
```
<a class="" href="hoge"><span class="">前の記事</span></a>
```
こうです！

Commit:
修正しました

qにクラスをつけました。
投稿本文にもpostTextというクラスをつけました。
author.pngのaltを山田太郎にしました。
ナビゲーションにaタグをつけました。
コピーライトのpにクラスをつけました。

Commit:
インデントの崩れ直しました

51から61までの崩れを直しました！
